crates/iota/src/client_commands.rs

```diff
@@ -149,22 +149,18 @@ pub enum IotaClientCommands {
         function: String,
         /// Type arguments to the generic function being called.
         /// All must be specified, or the call will fail.
-        #[arg(
-            long,
-            value_parser = parse_iota_type_tag,
-            num_args(1..),
-        )]
+        #[arg(num_args(1..))]
```

Comment:
Need to add the `!` to the PR description, bc this alone is a breaking change, no? Are we sure about removing the `long` attribute? :thinking:

Reply:
Good catch, this should actually stay, reverted in https://github.com/iotaledger/iota/pull/7881/commits/28cad187335eff3b58cc6ba2590536c370571815